src/Raven.Client/Documents/Session/AbstractDocumentQuery.cs

```diff
@@ -1250,7 +1250,7 @@ private void BuildInclude(StringBuilder queryText)
             if (DocumentIncludes.Count == 0 &&
                 HighlightingTokens.Count == 0 &&
                 ExplanationToken == null &&
-                QueryTimings == null &&
+                !ShouldIncludeTimings.Value &&
```

Comment:
Pretty sure that this won't work. Are you using the _same_ instance here across the board?
Because if I'm reading this properly, the instance you use and the internal instance aren't the same.

Reply:
I use the same instance, that's why I used 'FlagHolder' instead of bool,
when I clone DocumentQuery I copy it.

Reply:
I meant, the same instance of `QueryTimings`

Reply:
I don't want to use the same instance of 'QueryTimings', because every DocumentQuery should have its own QueryTiming.